solidity/config.yaml

```diff
@@ -1,4 +1,16 @@
+#Arguments to solc
+#_solcArgs:
+#Choose the number of epochs to use in coverage-guided testing
 _epochs: 2
+#Set the gas limit for each test
 _gasLimit: 0xfffff
+#Number of tests that will run for each property
 _testLimit: 10000
-_range: 10
\ No newline at end of file
+#Honestly not sure what this is
```

Comment:
https://github.com/trailofbits/echidna/blob/master/lib/Echidna/Exec.hs#L161 do you want to rename? if it's unclear to you, might be to users as well